tests/baselines/reference/strictOptionalProperties1.errors.txt

```diff
@@ -100,8 +111,8 @@ tests/cases/compiler/strictOptionalProperties1.ts(119,5): error TS2411: Property
     function f4a(t1: [number, string?], t2: [number, string?, string?]) {
         t1 = t2;  // Wasn't an error, but should be
         ~~
-!!! error TS2322: Type '[number, string?, string?]' is not assignable to type '[number, string?]'.
-!!! error TS2322:   Target allows only 2 element(s) but source may have more.
+!!! error TS2375: Type '[number, string?, string?]' is not assignable to type '[number, string?]' with exactOptionalPropertyTypes: true. Consider adding 'undefined' to the types of the target's properties.

```

Comment:
This error needs two fixes, and now we suggest one in the elaboration and the other as a codefix.

This is confusing because they appear to disagree, but I don't know that it's worth fixing.